openpype/hosts/houdini/plugins/publish/collect_cache_farm.py

```diff
@@ -0,0 +1,82 @@
+import os
+import pyblish.api
+import hou
+from openpype.hosts.houdini.api import lib, colorspace
+from openpype.hosts.houdini.api.lib import get_color_management_preferences
+
+
+class CollectDataforCache(pyblish.api.InstancePlugin):
+    """Collect data for caching to Deadline."""
+
+    order = pyblish.api.CollectorOrder + 0.1
+    families = ["ass", "pointcache",
```

Comment:
Regarding alembic and bgeo pointcache 
after the last update #4588 and #5324

I think we can use `pointcache` or we can use `abc` and `bgeo` separately.
however, if we used them separately a dependency may occur as it will be required to have this specific collector `collect_pointcache_type.py` to run before this collector. 
```suggestion
    families = ["ass", "abc",
```